test/tf/tests/assert_less_equal_test.cpp

```diff
@@ -25,6 +25,8 @@
 
 #include <tf_test.hpp>
 
+// Skip this test for libstdc++ debug for now since it exposes a bug in protobuf
+#ifndef _GLIBCXX_DEBUG
```

Comment:
Make this a todo and then make an issue for this if we're stubbing it out due to the libstdc++ tests

Reply:
I added an issue here:

https://github.com/ROCm/AMDMIGraphX/issues/4087

You can add more info if you want.